CONTRIBUTING.rst

```diff
@@ -223,7 +223,14 @@ syntax above so that the issue(s) will be closed upon merging the PR.
 PR Labels
 ~~~~~~~~~
 
-Add the labels that best describe your PR ('enhancement', 'documentation', etc...
+You **must** add labels to your PRs, as these are used to automatically generate Changelog:
+
+- **Category:** Choose **one** label that describes the `category <https://github.com/shimming-toolbox/shimming-toolbox-py/wiki/Label-definition#Category>`_ (purple labels).
+
+- **Shimming-toolbox Function:** Choose one or multiple labels corresponding to the files/functions that are mainly affected by the PR (light blue labels).
```

Comment:
for now, i suggest we remove those sub-labels (it's not like we have 50+ functions to maintain)
```suggestion

```